pkg/policy/id_test.go

```diff
@@ -77,8 +81,8 @@ func TestIDUnmarshalJSON(t *testing.T) {
 		{[]byte(`"foo"`), ID("foo"), false},
 		{[]byte(`"1234"`), ID("1234"), false},
 		{[]byte(`"DenyEncryptionSt1"`), ID("DenyEncryptionSt1"), false},
+		{[]byte(`"foo bar"`), ID("foo bar"), false},
```

Comment:
Same as above

Reply:
It's not unnoticed @balamurugana t.Errorf() simply continues to next test case. t.Fatalf is not correct.

Reply:
t.Errorf is also the most common used in tabled test cases when you to see all tests cases failing, there is no reason for us to fail only one test. We have no state or relation between test cases.

Reply:
Unable to understand.  Do you say `t.Errorf()` triggers aborting test function if any `t.Errorf()` occurred after completing all test cases? 
If not `t.Fatal()` is better which stops immediately on first failure.  So that its easy to look and fix when `make test` is run manually or in travis-ci; else a good amount of search is required.

Reply:
> nable to understand. Do you say t.Errorf() triggers aborting test function if any t.Errorf() occurred after completing all test cases?

Yes definitely.   t.Errorf is the most commonly used in a scenario where you are simply validating a bunch of tests, t.Fatalf is only used when you don't want the test to continue further like when you want some code to stop immediately. There is of course use of t.Fatalif in certain cases - but not in this case. 

One such example to look at https://golang.org/src/unicode/utf8/utf8_test.go

Reply:
Personally I prefer fast failure to know the issue immediately than later.  With this tiny test cases it might look OK but I wouldn't do.